config/default/filter.format.full_html.yml

```diff
@@ -55,7 +55,7 @@ filters:
     status: false
     weight: -43
     settings:
-      allowed_html: '<em> <strong> <cite> <blockquote cite> <code> <ul type> <ol start type> <li> <dl> <dt> <dd> <h2 id> <h3 id> <h4 id> <h5 id> <h6 id> <table class="decanter-table--borderless"> <caption> <tbody> <thead> <tfoot> <th> <td> <tr> <a href hreflang data-entity-substitution data-entity-type data-entity-uuid title class="decanter-button decanter-button--big decanter-button--secondary decanter-more-link decanter-external-link"> <drupal-entity data-entity-type data-entity-uuid data-entity-embed-display data-entity-embed-display-settings data-align data-caption data-embed-button> <p class="decanter-font-lead decanter-font-splash decanter-short-line-length decanter-caption decanter-credits"> <h1> <pre> <span class="emphasized-text"> <drupal-media data-entity-type data-entity-uuid data-align data-caption data-* alt>'
+      allowed_html: '<em> <strong> <cite> <blockquote cite> <code> <ul type> <ol start type> <li> <dl> <dt> <dd> <h2 id> <h3 id> <h4 id> <h5 id> <h6 id> <caption> <tbody> <thead> <tfoot> <th> <td> <tr> <drupal-entity data-entity-type data-entity-uuid data-entity-embed-display data-entity-embed-display-settings data-align data-caption data-embed-button> <h1> <pre> <table class="decanter-table--borderless hs-table--borderless"> <a href hreflang data-entity-substitution data-entity-type data-entity-uuid title class="decanter-button decanter-button--big decanter-button--secondary decanter-more-link decanter-external-link hs-button hs-button--big hs-secondary-button hs-more-link hs-external-link hs-file-link hs-pdf-link hs-private-link"> <hr> <drupal-media data-entity-type data-entity-uuid data-align data-caption data-* alt data-entity-type data-entity-uuid data-view-mode data-align data-caption alt title> <p class="decanter-font-lead decanter-font-splash decanter-short-line-length decanter-caption decanter-credits hs-font-lead hs-font-splash hs-short-line-length hs-breakout-box hs-caption hs-credits"> <span class="emphasized-text hs-emphasized-text">'
```

Comment:
The allowed attributes needs to be fixed.
Related to this issue: https://www.drupal.org/project/drupal/issues/2710427